src/main/groovy/com/google/appengine/task/endpoints/WebXmlProcessing.groovy

```diff
@@ -36,7 +36,28 @@ class WebXmlProcessing {
     }
 
     private static NodeChild getSystemServiceServlet(GPathResult root) {
-        def servlets = root."servlet".findAll { it."servlet-class".text().trim() == "com.google.api.server.spi.SystemServiceServlet" ? it : null }
+        def servlets = root."servlet".findAll { servlet ->
```

Comment:
Ah makes sense.  So there is another option here where you can specify `serviceClasses` instead of relying on auto detection.

Lets move this to another PR though.